Chefs.Api/Chefs.Api.csproj

```diff
@@ -10,4 +10,8 @@
 		<PackageReference Include="Microsoft.AspNetCore.OpenApi" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <ProjectReference Include="..\Chefs.Models\Chefs.Models.csproj" />
```

Comment:
This is gonna suck but we should follow the pattern from the recommended template and it should be Chefs.DataContracts :(

![image](https://github.com/user-attachments/assets/3eda4707-615e-4649-a9d8-9d47d51dabd7)